CONTRIBUTING.md

```diff
@@ -7,6 +7,9 @@ See the `10) Reporting a Windows crash (BSOD)` section below for detailed instru
 ## Reporting issues in Windows Console or WSL text rendering/user experience
 Note that WSL distro's launch in the Windows Console (unless you have taken steps to launch a 3rd party console/terminal). Therefore, *please file UI/UX related issues in the [Windows Console issue tracker](https://github.com/microsoft/console)*.
 
+## Reporting WSL Utilities (WSLU) issues
+If your issue pertains to one of the `wslact`, `wslclip`, `wslfetch`, `wslgsu`, `wslpath`, `wslsys`, `wslupath`, `wslusc`, `wslvar`, or `wslview` commands, *please file your issue in [the WSL Utilities repository](https://github.com/wslutilities/wslu) instead*.
```

Comment:
We already ship a wslpath. This would be confusing for us to put in our docs? Is this intentional?

Reply:
Sorry. I was completely confused about this. Please ignore my PR.